server/src/test/java/io/crate/analyze/CreateAlterTableStatementAnalyzerTest.java

```diff
@@ -953,28 +948,21 @@ public void testCreateTableWithInvalidColumnName() {
             .hasMessage("\"_test\" conflicts with system column pattern");
     }
 
-    @Test
-    public void testCreateTableShouldRaiseErrorIfItExists() {
```

Comment:
Duplicate of 

https://github.com/crate/crate/blob/d3673373f3ddf4b2fe2063ad05dc207ae2ba05a2/server/src/test/java/io/crate/integrationtests/DDLIntegrationTest.java#L133-L139

I removed the analzyer logic because we've two other places where the check happens, one is:

https://github.com/crate/crate/blob/d3673373f3ddf4b2fe2063ad05dc207ae2ba05a2/server/src/main/java/io/crate/execution/ddl/tables/TableCreator.java#L80

The other should be in the atomic cluster-state update